src/openssl/digest/digest_io.cr

```diff
@@ -0,0 +1,55 @@
+require "./digest_base"
+
+# Wraps an IO by calculating a specified digest on read and/or write operations
```

Comment:
It's amazing how fast docs can become outdated. Thanks for the attention to detail!